BottomSheetDemo/Sources/User Interface/Screens/Root/RootViewController.swift

```diff
@@ -54,7 +54,10 @@ final class RootViewController: UIViewController {
         let viewController = ResizeViewController(initialHeight: 300)
         presentBottomSheetInsideNavigationController(
             viewController: viewController,
-            configuration: .default
+            configuration: .default,
+            dismissCompletion: {
+                print("TWA: dismissCompletion")
+            }
```

Comment:
If this option is not needed let's just ignore it and remove `dismissCompletion`?